packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -24,10 +26,22 @@ const bootstrapNewUser = async (
   searchParams?: string
 ) => {
   const r = await getRethink()
-  const {id: userId, createdAt, preferredName, email, featureFlags, tier, segmentId} = newUser
+  const {
+    id: userId,
+    createdAt,
+    preferredName,
+    email,
+    featureFlags,
+    tier,
+    segmentId,
+    identities
+  } = newUser
   // email is checked by the caller
   const domain = email.split('@')[1]!
-  const isCompanyDomain = await dataLoader.get('isCompanyDomain').load(domain)
+  const [isCompanyDomain, organizations] = await Promise.all([
+    dataLoader.get('isCompanyDomain').load(domain),
+    dataLoader.get('organizationsByActiveDomain').load(domain)
```

Comment:
-1 commenting here, but could be fixed further down: We're trying to update the `activeDomain` of organizations only with company domains. But this is a moving target and was just recently updated. This means there are companies which have an `activeDomain` which is not a company domain. These will be returned here and we need to check `isCompanyDomain` somewhere before doing all the autojoin logic.